### PR TITLE
git guide for beginners

### PR DESCRIPTION
**Pull request by vyaspranjal33**

### CODE REVIEW

File: LearningX.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ This has useful links to learn X.
 4. [GitHub Help Guide](https://help.github.com/)
 5. [IRC](https://fedoramagazine.org/beginners-guide-irc/)
 6. [Git and Github in Plain English](https://red-badger.com/blog/2016/11/29/gitgithub-in-plain-english)
+7. [Git Beginner's Guide](https://backlog.com/git-tutorial/)
```
Comment (nikhita) [inline LearningX.md:9]: @vyaspranjal33 Thanks for your PR, but I don't think we want to include resources that are not free or are partially free.

**Comment by vyaspranjal33** [comment]
@nikhita  these are only guides ,it doesn't ask to buy any subscription or such , its blog post , which has articles on git
